recipes/openms-meta/build.sh

```
@@ -9,10 +9,196 @@ export CXXFLAGS=${CXXFLAGS//${USELESS}/}
+# Now we're in openms/THIRDPARTY
+
+if ! command -v curl &> /dev/null; then
+    echo "curl not found, installing..."
+    if command -v apt-get &> /dev/null; then
+        apt-get update && apt-get install -y curl
+    fi
+fi
+
+curl -L -o THIRDPARTY-release-3.4.0.tar.gz https://api.github.com/repos/OpenMS/THIRDPARTY/tarball/release/3.4.0
+tar -xzf THIRDPARTY-release-3.4.0.tar.gz
+cd OpenMS-THIRDPARTY-*/Linux/x86_64/
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)